src/js/components/Meter/doc.js

```diff
@@ -52,9 +52,13 @@ export const doc = Meter => {
       PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge']),
       PropTypes.string,
     ])
-      .description('The size of the Meter.')
+      .description(
+        `
+        The size of the Meter. 'full' with type="circle" creates a pie chart.
```

Comment:
Is thickness truly the "size of the Meter"? Also, I'm not sure what "full" means, it's not documented as part of the prop.

Reply:
oops, left over from an earlier approach, good catch, removed